sympy/physics/tests/test_secondquant.py

```diff
@@ -90,6 +91,7 @@ def test_operator():
 def test_create():
     i, j, n, m = symbols('i,j,n,m')
     o = Bd(i)
+    assert latex(Bd(i)) == "b^\\dagger_{i}"
```

Comment:
This could probably be `latex(o)`.

Reply:
@jksuom Yeah you're right. I got things mixed up but `latex(o)` should be possible since `o` is not a symbol. I'll fix that.